examples/kubernetes-config/kubernetes-clustersubnet.json

```diff
@@ -14,7 +14,7 @@
     },
     "agentPoolProfiles": [
       {
-        "name": "agent",
+        "name": "agentpool1",
```

Comment:
O.K., @dmitsh's recommendations makes sense. We'll have to fix the E2E configs to point to the new files then. You can do that easily thusly:

In the following three files:
- `test/acse-conf/acse-feature-validation.json`
- `test/acse-conf/acse-regression.json`
- `test/e2e/kubernetes-deployments.json`

Replace all `"cluster_definition"` values of `"custom-pod-cidr/kubernetes.json"` with the new relative filepath `"kubernetes-config/kubernetes-clustersubnet.json"`